__init__.py

```diff
@@ -59,13 +60,12 @@ def validate(self, tmpl):
 
 
 class License(zeekpkg.template.Feature):
-
     def license_keys(self, tmpl):
         licdir = os.path.join(tmpl.templatedir(), self.contentdir())
         return sorted(os.listdir(licdir))
 
     def contentdir(self):
-        return 'license'
+        return 'license-feature'
```

Comment:
I'm wondering whether we should expand this so it's consistent for all features. Maybe end all of their folder names in `-feature`, or put them all in `features/<name>/`?

Reply:
I added a patch to move all feature contentdirs to a `features/` subfolder.